### DISCUSSION TITLE
Confirm Page Refresh

### DISCUSSION THREAD

**Discussion by minecraftchest1** [jellyfin/jellyfin-web#3883] [Ideas]
Currently, if a user refreshes a page of the Jellyfin Web Client, their currently playing media, as well as up next and previously played list is lost. It would be nice if there was an option to presist this across refreshes, but a viviblle alternative would be to prompt the user before allowing the page to be refreshed.

**Comment by kimboslice99** [comment]
This is old but nonetheless

Prepend this to main.jellyfin.bundle.js

It will prompt are you sure if you click reload while playing a video

Issue is, if you click cancel it will fail and the enitre UI becomes unresponsive, maybe someone more familiar with jellyfin can explain how to stop that
```
window.addEventListener('beforeunload', function (e) {
    if(window.location.hash == "#!/video"){
        e.preventDefault();
        return e.returnValue = "";
    }
});
```
